### PR TITLE
Add WebhookPayloadTemplate class to handle curly braces webhook syntax

### PR DESCRIPTION
**Pull request by mnmkng**
Title says all. Putting it here since it's gonna be useful in SDK too when creating webhooks programmatically.

Currently does not support dot notation `{{var.prop}}`, but I'll add it later.

### CODE REVIEW

File: src/webhook_payload_template.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
+const { JsonToken, jsonStringifyExtended } = require('./utilities.client');
```
Comment (jancurn) [inline src/webhook_payload_template.js]: Wasn't this supposed to be called `JsonVariable` ?

File: src/webhook_payload_template.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+     *     "bar": ["bar"],
+     *     "res": {{resource}},
+     * }
+     * ```
+     *
+     * @param {string} variable
+     * @return {JsonToken}
+     */
+    static getVariable(variable) {
+        return new JsonToken(variable);
```
Comment (jancurn) [inline src/webhook_payload_template.js]: Is it really necessary to define such a simple function for this?

File: src/webhook_payload_template.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+ *
+ * This is useful to create dynamic webhook payloads where a template is saved on webhook
+ * creation and then variables are dynamically added on webhook dispatch.
+ *
+ * **Example:**
+ * ```js
+ * const payloadTemplate = `{
+ *    "id": "some-id",
+ *    "createdAt": "2019-05-08T15:22:21.095Z",
+ *    "dataToSend": {{data}}
```
Comment (jancurn) [inline src/webhook_payload_template.js]: From the comments it's not clear how to handle hierarchical objects, is it even possible? E.g. to use `{{data.status}}`

File: src/webhook_payload_template.js (hunk 1 above)
Comment (mnmkng) [reply src/webhook_payload_template.js]: Don't know, that was @mtrunkat's PR. I'm just using it.

File: src/webhook_payload_template.js (hunk 2 above)
Comment (mnmkng) [reply src/webhook_payload_template.js]: It's an encapsulation thing. You don't want to do

```
const { jsonToken, jsonStringifyExtended } = require('apify-shared/utilities.client');
const WebhookPayloadTemplate = require('apify-shared/webhook_payload_template');

const var = new jsonToken('var');
```

when you can just think about having the one class and not even know about the `jsonToken` which is something else entirely.

File: src/webhook_payload_template.js (hunk 3 above)
Comment (mnmkng) [reply src/webhook_payload_template.js]: It's not possible and it's in the OP of the PR.

File: src/webhook_payload_template.js (hunk 1 above)
Comment (mtrunkat) [reply src/webhook_payload_template.js]: I renamed that to `JsonVariable`

**Pull request merged**